README.md

```diff
@@ -108,6 +108,20 @@ And install it with:
 
 `ansible-galaxy collection install ./theforeman-foreman-*.tar.gz`
 
+## Running tests
```

Comment:
We have https://github.com/theforeman/foreman-ansible-modules/blob/develop/docs/testing.md for that?

Reply:
Damn I knew it it is somewhere...